apps/platform-app/src/app/[locale]/(profile)/(routes)/my-profile/(components)/CreateProfile/PersonalInfo/PersonalInfo.tsx

```diff
@@ -11,6 +11,7 @@ import { useFormikContext } from 'formik'
 import { useSession } from 'next-auth/react'
```

Comment:
**praise**: Good job integrating the CVUploader component into the PersonalInfo section. This enhances the user profile creation process.